OutOfSchool/OutOfSchool.WebApi/Controllers/V1/SupportInformationController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using OutOfSchool.Common.PermissionsModule;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services;
+
+namespace OutOfSchool.WebApi.Controllers.V1
+{
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]/[action]")]
+    public class SupportInformationController : ControllerBase
+    {
+        private readonly ISupportInformationService supportInformationService;
+        private readonly ILogger<SupportInformationController> logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SupportInformationController"/> class.
+        /// </summary>
+        /// <param name="supportInformationService">Service for SupportInformation model.</param>
+        /// <param name="logger"><see cref="Microsoft.Extensions.Logging.ILogger{T}"/> object.</param>
+        public SupportInformationController(
+            ISupportInformationService supportInformationService,
+            ILogger<SupportInformationController> logger)
+        {
+            this.supportInformationService = supportInformationService ?? throw new ArgumentNullException(nameof(supportInformationService));
```

Comment:
When Asp.Net Core tries to activate this controller, it will throw new InvalidOperationException if unable to resolve some services from DependencyInjection.ActivatorUtilities.GetService() in our cases.

Reply:
OK